tests/TestBaseClass.php

```diff
@@ -73,6 +73,8 @@ protected static function importSurvey($fileName)
         }
 
         // Reset the cache to prevent import from failing if there is a cached survey and it's active.
+        // When importing, activating, deleting and importing again (usual with automated tests), 
```

Comment:
OK, it can happen too in real situation … (i just have it this morning) but still rare issue ;)